tests/unit/repo/test_apt_key_manager.py

```diff
@@ -256,13 +256,19 @@ def test_is_key_installed_with_gpg_failure(
     mock_logger.warning.assert_called_once_with("gpg error: some error")
 
 
+@pytest.mark.parametrize("key_id", (None, "FAKE-KEY-ID-FROM-GNUPG"))
```

Comment:
@lengau I assumed you meant a unit test in your request; let me know if I was wrong (there is an integration test that does this key-id/no key-id parametrization already)

Reply:
Exactly what I meant, thank you! Looks great!